ruma-api-macros/src/derive_outgoing/wrap_incoming.rs

```diff
@@ -0,0 +1,58 @@
+use syn::{
```

Comment:
So we won't need this whole module and everything based on it anymore. We're still going to support `#[wrap_incoming]` without arguments and `#[incoming_no_deserialize]`, but that's it.

Let's talk on Matrix about the new transformations we need.